### PR TITLE
New ports added to the code

### PR DESCRIPTION
**Pull request by costinm**
Update the doc for ports in use to reflect reality.

The HBONE port is not implemented yet, but we are starting work, better to reserve it.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/deployment/requirements/index.md [hunk 1]
```diff
@@ -94,11 +94,15 @@ The following ports and protocols are used by the Istio sidecar proxy (Envoy).
...
 | 15000 | TCP | Envoy admin port (commands/diagnostics) | Yes |
 | 15001 | TCP | Envoy outbound | No |
+| 15002 | HBONE | Reserved for HBONE | No |
+| 15004 | HTTP | Debug port | Yes |
 | 15006 | TCP | Envoy inbound | No |
 | 15008 | TCP | Envoy tunnel port (inbound) | No |
 | 15020 | HTTP | Merged Prometheus telemetry from Istio agent, Envoy, and application | No |
 | 15021 | HTTP | Health checks | No |
 | 15090 | HTTP | Envoy Prometheus telemetry | No |
+| 15053 | DNS  | DNS port, if capture is enabled | Yes |
```
Comment (ericvn) [inline content/en/docs/ops/deployment/requirements/index.md:104]: If keeping the ports in order, this port is not.

File: content/en/docs/ops/deployment/requirements/index.md [hunk 2]
```diff
@@ -94,11 +94,15 @@ The following ports and protocols are used by the Istio sidecar proxy (Envoy).
...
+
```
Comment (ericvn) [inline content/en/docs/ops/deployment/requirements/index.md:105]: The linter will also complain about the extra blank line being added.

**Comment by costinm** [comment]
Fixed, also reordered the 443/8080 if we want ports ordered.

Updated the text a bit for the plaintext ports to make it a bit more clear they are intended for 'secure networks',
and added the container port for webhooks which was missing.

File: content/en/docs/ops/deployment/requirements/index.md [hunk 3]
```diff
@@ -94,21 +94,25 @@ The following ports and protocols are used by the Istio sidecar proxy (Envoy).
 |----|----|----|----|
 | 15000 | TCP | Envoy admin port (commands/diagnostics) | Yes |
 | 15001 | TCP | Envoy outbound | No |
+| 15004 | HTTP | Debug port | Yes |
 | 15006 | TCP | Envoy inbound | No |
-| 15008 | TCP | Envoy tunnel port (inbound) | No |
+| 15008 | H2 | HBONE mTLS tunnel port | No |
+| 15009 | H2C | HBONE port for secure networks | No |
```
Comment (kyessenov) [inline content/en/docs/ops/deployment/requirements/index.md:100]: This is not ready, right? Why are we publishing it already.

Comment (costinm) [reply content/en/docs/ops/deployment/requirements/index.md:100]: 15008 was already published - I just updated the doc.

And we are starting to use them ( via bootstrap or envoy filter ) - should be reserved and documented.

**Review by ericvn — APPROVED** [review]

**Pull request merged**